skiko/src/nativeMain/kotlin/org/jetbrains/skia/Actuals.native.kt

```diff
@@ -6,7 +6,7 @@ actual fun <R> commonSynchronized(lock: Any, block: () -> R) {
     block()
 }
 
-actual fun String.intCodePoints(): IntArray = TODO()
+actual fun String.intCodePoints(): IntArray = toList().map { it.code }.toIntArray()
```

Comment:
Seems to be sensitive performance-wise, could we make it more optimal? I.e.
```
IntArray(this.length) { it -> this[it].code }
```

Reply:
can not agree more, pushed